dask/dataframe/multi.py

```diff
@@ -295,7 +295,34 @@ def merge_chunk(
                 args[0] = "inner"
             else:
                 kwargs["how"] = "inner"
-    out = lhs.merge(rhs, *args, **kwargs)
+    if len(args) and args[0] == "leftanti" or kwargs.get("how", None) == "leftanti":
+        left = None
+        right = None
+
+        left_on = kwargs.get("left_on", None)
+        right_on = kwargs.get("right_on", None)
+
+        if isinstance(left_on, list):
+            left_on = left_on[0]
+        if isinstance(right_on, list):
+            right_on = right_on[0]
```

Comment:
I'm not sure I understand why it's okay to ignore all but the first element here?

Reply:
Digging into this more, it's not immediately clear how we could handle the multi-columnar case right now, as the behavior of `isin` in this block switches depending on whether the operands are series or dataframes; for example, considering the example data:

```python
(Pdb) emp
   emp_id   name city  salary
0     101   John  Cal   50000
1     102    Tom  Mum   40000
2     103  Harry  Del   80000
3     104  Rahul  Ban   60000
4     105  Sakil  Del   90000
(Pdb) skills
   skill_id  emp_id skill_name
0       404     103       Dask
1       405     101      Spark
2       406     105          C
3       407     102     Python
4       408     101          R
(Pdb) emp.emp_id.isin(skills.emp_id)
0     True
1     True
2     True
3    False
4     True
Name: emp_id, dtype: bool
(Pdb) emp[["emp_id"]].isin(skills[["emp_id"]])
   emp_id
0   False
1   False
2   False
3   False
4   False
```

Could think of a few hacky ways to do this with an apply function, but not sure if any of these are efficient enough to consider for now - does it make sense to add this functionality and guard against multi-column left anti joins on CPU, or should we just consider this blocked for now?